content/2020/2020-11/2020-11-27-pulsar-weekly.md

```diff
@@ -0,0 +1,151 @@
+
+### Notable Bug Fix
+
+- [Broker] Use `Awaitility` instead `Thread.sleep` in `InactiveTopicDeleteTest.java`.
+
+    https://github.com/apache/pulsar/pull/8647 (@[315157973](https://github.com/315157973))
+
+- [Broker] Fix the Apache rat check issue.
+
+    https://github.com/apache/pulsar/pull/8665 (@[codelipenghui](https://github.com/codelipenghui))
+
+- [Broker] Clear delayed messages when clearing the backlog.
+
+    https://github.com/apache/pulsar/pull/8691 (@[codelipenghui](https://github.com/codelipenghui))
+
+- [Tiered Storage] Fix the Jclouds Azure credential error.
+
+    https://github.com/apache/pulsar/pull/8693 ([@gaoran10](https://github.com/gaoran10))
+
+- [Build] Update the management module version.
+
+    https://github.com/apache/pulsar/pull/8695 ([@gaoran10](https://github.com/gaoran10))
+
+- [Build] Upgrade the license-maven-plugin to version 4.0.rc2.
+
+    https://github.com/apache/pulsar/pull/8706 ([@lhotari](https://github.com/lhotari))
+
+- [Build] Fix the issue in Maven build that re-compiles all classes.
+
+    https://github.com/apache/pulsar/pull/8707 ([@lhotari](https://github.com/lhotari))
+
+- [Build] Specify the `maven-antrun-plugin` version and migrate it to use the newer syntax.
+
+    https://github.com/apache/pulsar/pull/8708 ([@lhotari](https://github.com/lhotari))
+
+- [CI] Fix the the issue that the "CI - Build - Multiple - OS" fails on the `macos-latest` runner.
+
+    https://github.com/apache/pulsar/pull/8713 ([@lhotari](https://github.com/lhotari))
+
+- [Java Client] Include `com.google.guava:failureaccess` in the shaded JAR.
+
+    https://github.com/apache/pulsar/pull/8697 ([@cimura](https://github.com/cimura))
+
+- [Prometheus Metrics] Update the PrometheusMetrics on binaries when building from the source tarball.
+
+    https://github.com/apache/pulsar/pull/8702 (@[eolivelli](https://github.com/eolivelli))
+
+- [Log Conf] Delete the explicit claim for the log configuration file position in surefire.
+
+    https://github.com/apache/pulsar/pull/8715 ([@Renkai](https://github.com/Renkai))
+
+- [WebSocket] Fix the initial sequence ID error.
+
+    https://github.com/apache/pulsar/pull/8724 ([@BewareMyPower](https://github.com/BewareMyPower))
+
+- [Java Client] Fix the warning log on the producer side when duplicated message is dropped.
+
+    https://github.com/apache/pulsar/pull/8729 ([@massakam](https://github.com/massakam))
+
+- [Unit Test] Fix the `MessagePublishBufferThrottleTest` flaky-test.
+
+    https://github.com/apache/pulsar/pull/8733 (@[315157973](https://github.com/315157973))
+
+- [Unit Test] Fix the `TopicsConsumerImplTest.testConsumerUnackedRedelivery` flaky-test.
+
+    https://github.com/apache/pulsar/pull/8738 (@[315157973](https://github.com/315157973))
+
+- [Tiered Storage] Initialize the offload manager only for one time.
+
+    https://github.com/apache/pulsar/pull/8739 ([@gaoran10](https://github.com/gaoran10))
+
+### Event / News
+
```

Comment:
Please add Pulsar Summit Asia 2020.
You can consult Dianjin or Jipei on the video and slides links.